### ISSUE TITLE
muongun - cubature.c lack code coverage / tests (Trac #1375)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1367]
http://software.icecube.wisc.edu/coverage/00_LATEST/MuonGun/private/cubature/

If a file is important enough to import from another project into our code base, it's important enough to have tests and coverage applied to it.

Migrated from https://code.icecube.wisc.edu/ticket/1375
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:14:07", 
    "description": "http://software.icecube.wisc.edu/coverage/00_LATEST/MuonGun/private/cubature/\n\nIf a file is important enough to import from another project into our code base, it's important enough to have tests and coverage applied to it.", 
    "reporter": "nega", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1458335647931556", 
    "component": "combo simulation", 
    "summary": "muongun - cubature.c lack code coverage / tests", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-10-01T13:59:56", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20151001T14:41:17: jvansanten commented:

> In changeset 138145:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="138145"
> Add a test for integration. See #1367
> ```

**Comment by icecube-trac** [comment]
Trac update at 20151014T12:59:47: 
* jvansanten changed resolution from "" to "fixed"
* jvansanten changed status from "new" to "closed"
* jvansanten commented:

> In changeset 138524:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="138524"
> We don't use 1D integration from cubature. Protect the entry point with an
> assert() to keep coverage stats happy. Fixes #1367.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:07: 
* olivas changed milestone from "November 2015" to ""
* olivas commented:

> Milestone November 2015 deleted
